.rspec

```diff
@@ -1 +1 @@
---require spec_helper
+--require spec_helper --tag ~smoke_test
```

Comment:
Am just curious, did this work? And mean that the smoke tests weren't run by default?

Reply:
yep, smoke tests are excluded, really neat solution thanks for that!